app/frontend/stylesheets/components/_navbar.scss

```diff
@@ -0,0 +1,42 @@
+
+.navbar {
+  margin-left: 300px;
+  display: flex;
+  // justify-content: space-around;
+  align-items: center;
+}
+
+.logo {
+  margin-right: 120px;
+}
+
+ul {
+  margin: auto;
+  display: flex;
+  justify-content: space-around;
+  max-width: 400px;
+}
+
+ul li {
```

Comment:
In general I'm a friend of using unique named div class containers if needed and adressing them in the scss, to modify their design. If using ul li p h1 and so on in a navbar.scss it's possible, that it will affect ul li p h1 outside of the _navbar.html.erb. 

Just a proposal.

Reply:
I agree with Stephan, this is not safe
But instead of giving them all a class, just embed everything inside the .navbar selector
That way it ensures that every propreties you apply to these element only apply if they are inside the navbar

.navbar {
   ...
   
   .logo {
   }
   
   ul {
     li {
       &:not(:first-child)::before {
       }
       
       a {
       }
     }
   }
 }

Reply:
Yes, this is nice. Looks good. If you do it like this, and you have nothing left, which is not nested in the .navbar you could move the whole content theoretically to application.scss. :)

Reply:
Yes i get your point guys. I just nested everything inside the navbar div and  now they don't affect anything else out of the navbar. Tested already!